TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -0,0 +1,598 @@
+//
+//  HomeOptionsCard.swift
+//  TCAT
+//
+//  Created by Omar Rasheed on 3/23/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+import DZNEmptyDataSet
+import GoogleMaps
+import SnapKit
+
+protocol HomeOptionsCardDelegate {
+    func updateSize()
+    func getCurrentLocation() -> CLLocation?
+}
+
+class HomeOptionsCardViewController: UIViewController {
+    
+    var delegate: HomeOptionsCardDelegate?
+    
+    var currentLocation: CLLocation? { return delegate?.getCurrentLocation() }
+    var tableView: UITableView!
+    var searchBar: UISearchBar!
+    let infoButton = UIButton(type: .infoLight)
+    var searchBarSeparator: UIView!
+    var timer: Timer?
+    var searchResultsSection: Section!
+
+    var loadingIndicator: LoadingIndicator?
+    var isLoading: Bool { return loadingIndicator != nil }
+    var isNetworkDown = false
+    let searchBarHeight: CGFloat = 54
+    let searchBarSeparatorHeight: CGFloat = 1
+    let headerHeight: CGFloat = 42
+    let tableViewRowHeight: CGFloat = 50
+    
+    var maxCardHeight: CGFloat {
```

Comment:
Will this comment still show up with a control click? If not, put it above the variable name so it will!